tests/ci/run_cross_tests.sh

```diff
@@ -61,7 +61,7 @@ echo "Testing AWS-LC shared library for ${TARGET_CPU}."
 for BO in "${BUILD_OPTIONS[@]}"; do
   run_build -DCMAKE_TOOLCHAIN_FILE="${SCRATCH_FOLDER}/${TARGET_CPU}.cmake" ${BO}
 
-  shard_gtest "${BUILD_ROOT}/crypto/crypto_test --gtest_also_run_disabled_tests"
+  shard_gtest "${BUILD_ROOT}/crypto/crypto_test"
```

Comment:
This "disabled" test only failed after running for ~1500 seconds (= 90 minutes).
```
[  FAILED  ] ECTest.DISABLED_ScalarBaseMultVectorsTwoPoint (1498091 ms)
```

The test is disabled due to it being slow.  When running the tests under emulation, as we do when cross-compiling, they run even slower.